src/Graph/Graph/Microsoft.Graph.psd1

```diff
@@ -18,7 +18,7 @@ ModuleVersion = '1.9.6'
 CompatiblePSEditions = 'Core', 'Desktop'
 
 # ID used to uniquely identify this module
-GUID = 'ba3e9757-d74f-45ea-9d0f-0287d0d45b9a'
+GUID = 'd4a04cff-ead8-4cbb-91da-87472929d860'
```

Comment:
@peombwa , is it expected that we change the module id during the build process? Anyone with a dependency on this id will be broken if we do that. Seems like in general this file should be static or at least generated from some other static artifact.